docs/data/date-pickers/date-time-field/BasicDateTimeField.tsx

```diff
@@ -1,15 +1,12 @@
 import * as React from 'react';
-import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
 import { LocalizationProvider } from '@mui/x-date-pickers';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { DateTimeField } from '@mui/x-date-pickers/DateTimeField';
 
 export default function BasicDateTimeField() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <DemoContainer components={['DateTimeField']}>
-        <DateTimeField label="Basic date time field" />
-      </DemoContainer>
+      <DateTimeField label="Basic date time field" />
```

Comment:
This and `BasicDateTimePicker` benefit from the `DemoContainer` to provide the necessary `minWidth`. 🙈 
![Screenshot 2023-02-15 at 15 48 09](https://user-images.githubusercontent.com/4941090/219045241-eee3b30d-e38c-47a0-a191-fc97ab1a1689.png)

Reply:
Arf, OK. I'm giving up on these changes then. It's too demanding for the bandwidth that I can allocate to it. I will keep it simple 😁